test/controllers/update/update.interrupt.card.controller.spec.ts

```diff
@@ -0,0 +1,53 @@
+jest.mock("ioredis");
```

Comment:
Check sonarqube for missing test coverage, just need one more test to cover all the conditions